packages/devtools_app/test/test_infra/flutter_test_driver.dart

```diff
@@ -345,25 +345,25 @@ class FlutterRunTestDriver extends FlutterTestDriver {
 
       // Map to WS URI.
       vmServiceWsUri =
-          convertToWebSocketUrl(serviceProtocolUrl: vmServiceWsUri);
+          convertToWebSocketUrl(serviceProtocolUrl: vmServiceWsUri!);
```

Comment:
Copy `vmService` and `vmServiceWsUri` into locals to avoid the extra null assertions.

Reply:
removed assertions